bigquery.js

```diff
@@ -0,0 +1,94 @@
+var BigQueryClient = require('google-cloud/node_modules/@google-cloud/bigquery');
+
+const projectData = require('./auth.json');
+
+const config = {
+  projectId: projectData['project_id'],
+  keyFilename: './auth.json'
+};
+
+const ds = 'atomic';
+
+const PARTITION_FIELDS = ['updated_at', 'created_at', 'fired_at', 'ts'];
+
+var BigQuery = function() {
+  this.conn = BigQueryClient(config);
+};
+
+BigQuery.prototype.insertInto = function(tableName, rows, options) {
+  //Here I should get the table based on name
+  // and insert in the right partition whenever the table is partitioned
+  var me = this;
```

Comment:
quando se usa Arrow function a função assume o contexto do lugar onde ela foi chamada. Como você usou `(row) => { ...` lá embaixo não precisaria do `me.table`. Poderia ficar `this.table`.

Vou tentar dar um exemplo tosco:

```javascript
let diego = {
  name: 'Diego',
  work: {
    name: 'Bidu',
    toString: function(callback) {
      callback();
    }
  },
  toString: function() {
    this.work.toString(function() {
      console.log(this.name);
    });
  }
}

diego.toString();

//undefined
```

com a Arrow function (linha que tem o `this.work.toString`:

```javascript
let diego = {
  name: 'Diego',
  work: {
    name: 'Bidu',
    toString: function(callback) {
      callback();
    }
  },
  toString: function() {
    this.work.toString(() => console.log(this.name) );
  }
}

diego.toString();
// Diego
```

Viu que nem precisou de um `var me = this`?

Reply:
Tem razão; não sei porque não funcionou antes, mas testei agora e deu certo, @helioalb https://github.com/Bidu/snowplow-bigquery-lambda-loader/pull/3/commits/5255512c7010720b3875e4371df0101f043a9987